spec/integration/category/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
```

Comment:
- [ ] You have integration specs, but a you are yet to add the unit tests as requested by the previous reviewer. Kindly add them.